contrib/node/src/python/pants/contrib/node/tasks/node_task.py

```diff
@@ -17,13 +19,17 @@
 from pants.contrib.node.targets.node_test import NodeTest
 
 
+logger = logging.getLogger(__name__)
```

Comment:
Tasks should use `self.context.log.debug(..)`, which puts their log messages in the right scope. Should delete this.